lib/ansible/modules/network/vyos/vyos_facts.py

```diff
@@ -27,6 +27,7 @@
 author:
   - Nathaniel Case (@qalthos)
   - Nilashish Chakraborty (@Nilashishc)
+  - Rohit Thakur (@rohitthakur2590)
 extends_documentation_fragment: vyos
 notes:
   - Tested against VyOS 1.1.8
```

Comment:
We need to specify which connection type the module works with and also point to platform guide.